x-pack/test/stack_functional_integration/configs/config.stack_functional_integration_base.js

```diff
@@ -16,10 +16,17 @@ const log = new ToolingLog({
   level: 'info',
   writeTo: process.stdout,
 });
+log.info(`REPO_ROOT = ${REPO_ROOT}`);
 log.info(`WORKSPACE in config file ${process.env.WORKSPACE}`);
-const stateFilePath = process.env.WORKSPACE
-  ? `${process.env.WORKSPACE}/qa/envvars.sh`
-  : `${REPO_ROOT}/../integration-test/qa/envvars.sh`;
+
+const INTEGRATION_TEST_ROOT = process.env.WORKSPACE
+  ? process.env.WORKSPACE
+  : `${REPO_ROOT}/../integration-test`;
+log.info(`integrationTestRoot = ${INTEGRATION_TEST_ROOT}`);
+
+// const stateFilePath = resolve(integrationTestRoot, '/qa/envvars.sh');
```

Comment:
Can we remove it?

Reply:
I wanted to use that line but it didn't work so instead I used;

```
const stateFilePath = `${INTEGRATION_TEST_ROOT}/qa/envvars.sh`;
```
Is there a preference?

Reply:
`resolve()` is resolving a sequence of paths to an absolute one, I don't see any benefits in using here.

Reply:
fixed

Reply:
I have the additional complication of running tests locally on Windows where paths have backslashes.  I was hoping resolve would help resolve the differences with things like `C:\path` joined with `../integration-test` (it doesn't actually work as well as I had hoped).